rules/downgrade/tests/Rector/Property/TypedPropertyRector/Fixture/classname_nullable_type.php.inc

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Rector\Downgrade\Tests\Rector\Property\TypedPropertyRector\Fixture;
+
+class ClassNameNullableTypeClass {
+    private ?\Rector\Downgrade\Tests\Rector\Property\TypedPropertyRector\Fixture\AnotherClassNameNullableTypeClass $property;
+}
+
+class AnotherClassNameNullableTypeClass {
+}
```

Comment:
These extra classes, that should be autolaoded but do not change at all, should be located in `/Source` directory, on the same level as `/Fixtures`

Reply:
Changed